src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\EcontBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('answear_econt');
+
+        $treeBuilder->getRootNode()
+            ->children()
+                ->scalarNode('user')->cannotBeEmpty()->end()
+                ->scalarNode('password')->cannotBeEmpty()->end()
```

Comment:
Technically it is a BC break, but given everything API provided will not be working anyway let's merge this as is and release as next minor normally